### ISSUE TITLE
Cannot drag&drop images from another drives mounted at /mnt

### ISSUE THREAD

**Issue by maboleth** [flathub/com.viber.Viber#19]
I'm sure this is due to flatpak nature of seeing things. However there's a setting that enables flatpak environment to take into consideration all drives mounted on /mnt

Please fix this in future versions.

**Comment by TingPing** [comment]
You can either give up security: `flatpak override --user --filesystem=host com.viber.Viber`

Or just accept it.

**Comment by maboleth** [comment]
> You either grant filesystem access: `flatpak override --user --filesystem=host com.viber.Viber` (You can be more granular and just do `/mnt/Foo`)
> 
> Or just accept it.

So you cannot change anything like GIMP team did for its flatpak, as the program now recognizes all drives, including phones?
I'm no programmer but to me these sandbox problems sound very similar.

**Comment by TingPing** [comment]
Yes but filesystem access is a privilege, it bypasses all possible security.

**Comment by maboleth** [comment]
> You either grant filesystem access: `flatpak override --user --filesystem=host com.viber.Viber` (You can be more granular and just do `/mnt/Foo`)

When I do that, is that temporary, do I have to type that every time I start viber? If so I should make a script.

**Comment by TingPing** [comment]
It is persistent.

**Comment by maboleth** [comment]
> It is persistent.

Wow, that was ultra fast! :D Thanks!

**Comment by akurtakov** [comment]
I know it's ancient but do you still face the problem?
